### PR TITLE
fix(pyright): use pyrightconfig.json like sign of root directory 

### PR DESCRIPTION
**Pull request by defntvdm**

### CODE REVIEW

File: lua/lspconfig/pyright.lua [hunk 1]
```diff
@@ -13,6 +13,7 @@ local root_files = {
   'setup.cfg',
   'requirements.txt',
   '.git',
+  'pyrightconfig.json',
```
Comment (lithammer) [inline lua/lspconfig/pyright.lua:16]: Not that it matters much because it's a breadth-first search, but I think you should put it before `.git`.

**Comment by lithammer** [comment]
You're gonna have to update the commit message to make commitlint happy.

https://github.com/conventional-changelog/commitlint/#what-is-commitlint

File: lua/lspconfig/pyright.lua (hunk 1 above)
Comment (defntvdm) [reply lua/lspconfig/pyright.lua:15]: thanks, done :)

**Comment by defntvdm** [comment]
Should I squash commits and rename it like PR?

**Comment by lithammer** [comment]
Thanks!

**Pull request merged**